docs/content/authorization-and-openfga.mdx

```diff
@@ -11,40 +11,34 @@ import { DocumentationNotice, Playground, ProductName, ProductNameFormat } from
 
 <DocumentationNotice />
 
-This section explains [authorization](#authentication-vs-authorization), [fine-grained authorization](#what-is-fine-grained-authorization-fga), [RBAC](#what-are-role-based-access-control-rbac-and-attribute-based-access-control-abac), [ABAC](#what-are-role-based-access-control-rbac-and-attribute-based-access-control-abac), [ReBAC](#what-is-relationship-based-access-control-rebac).
+OpenFGA reies on several understandings of [authorization](#authentication-vs-authorization), including [fine-grained authorization](#what-is-fine-grained-authorization-fga), [role-based access control](#what-are-role-based-access-control-rbac-and-attribute-based-access-control-abac), [attribute-based access control](#what-are-role-based-access-control-rbac-and-attribute-based-access-control-abac), and [relationship-based access control](#what-is-relationship-based-access-control-rebac).
 
 ## What Is OpenFGA?
 
-**OpenFGA** is an open source solution to [Fine-Grained Authorization](#what-is-fine-grained-authorization-fga) that applies the concept of [ReBAC](#what-is-relationship-based-access-control-rebac). It was created by the [Okta FGA](https://docs.fga.dev/) team and was inspired by [Zanzibar](#what-is-zanzibar). It was designed for reliability and low latency at a high scale. It offers an HTTP API and has SDKs for programming languages including [Node.js/JavaScript](https://www.npmjs.com/package/@openfga/sdk), [GoLang](https://github.com/openfga/go-sdk), [.NET](https://www.nuget.org/packages/OpenFga.Sdk), [Python](https://pypi.org/project/openfga-sdk), [Java](https://central.sonatype.com/artifact/dev.openfga/openfga-sdk). More SDKs and integrations such as Rego are planned for the future.
+**OpenFGA** is an open source solution to [Fine-Grained Authorization](#what-is-fine-grained-authorization-fga), or **FGA**, that applies the concept of [relationship-based access control](#what-is-relationship-based-access-control-rebac), or **ReBAC**. It was created by the [Okta FGA](https://docs.fga.dev/) team and inspired by [Zanzibar](#what-is-zanzibar). Designed for reliability and low latency at a high scale, FGA offers an HTTP API and SDKs for programming languages like [Node.js/JavaScript](https://www.npmjs.com/package/@openfga/sdk), [GoLang](https://github.com/openfga/go-sdk), [.NET](https://www.nuget.org/packages/OpenFga.Sdk), [Python](https://pypi.org/project/openfga-sdk), and [Java](https://central.sonatype.com/artifact/dev.openfga/openfga-sdk), with additional SDKs and integrations (including Rego) planned for future releases.
```

Comment:
```suggestion
**OpenFGA** is an open source solution to [Fine-Grained Authorization](#what-is-fine-grained-authorization-fga), or **FGA**, that applies the concept of [relationship-based access control](#what-is-relationship-based-access-control-rebac), or **ReBAC**. It was created by the [Okta FGA](https://docs.fga.dev/) team and inspired by [Zanzibar](#what-is-zanzibar). Designed for reliability and low latency at a high scale, OpenFGA offers an HTTP and GRPC APIs and SDKs for programming languages like [Node.js/JavaScript](https://www.npmjs.com/package/@openfga/sdk), [GoLang](https://github.com/openfga/go-sdk), [.NET](https://www.nuget.org/packages/OpenFga.Sdk), [Python](https://pypi.org/project/openfga-sdk), and [Java](https://central.sonatype.com/artifact/dev.openfga/openfga-sdk), with additional SDKs and integrations [planned](https://github.com/orgs/openfga/projects/1) for future releases.
```

Reply:
>OpenFGA offers an HTTP and GRPC APIs and SDKs

this portion doesn't work exactly properly. How about:

"OpenFGA offers both HTTP and GRPC APIs, as well as SDKs"

Reply:
Yep!